src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,53 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+
 public class FileWork {
+    private static final char SPECIFIED_CHARACTER = 'w';
+
+    private static final String UNUSED_CHARACTERS = "[\\.,?!]";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        BufferedReader bufferedReader = null;
+        StringBuilder stringBuilder;
+        try {
+            bufferedReader = new BufferedReader(new FileReader(fileName));
+            stringBuilder = new StringBuilder();
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                stringBuilder.append(value).append(System.lineSeparator());
```

Comment:
can we process data right when we read the line?

Reply:
No, because if we get an empty file, method .toLowerCase crashed the program